scripts/halium

```diff
@@ -57,6 +57,18 @@ identify_boot_mode() {
 		esac
 	fi
 
+	# On Android 8+ devices the 'android' boot mode is broken and should be avoided.
+	# This behavior can be overridden with the cmdline flag 'halium_no_avoid_android_mode'
+	if ! grep -q halium_no_avoid_android_mode /proc/cmdline; then
+		sdk_version=$(grep 'ro.build.version.sdk' /android-system/build.prop | cut -d'=' -f2) # e.g. 26 for Android 8.0
+		[ -z "$sdk_version" ] && sdk_version=0
+		tell_kmsg "Android image SDK version: $sdk_version"
+		if [ "$BOOT_MODE" = "android" ] && [ $sdk_version -ge 26 ]; then
```

Comment:
Maybe refers to Android's API level page? (https://source.android.com/setup/start/build-numbers)

Reply:
Yes, that's exactly what this is. Should I rename things to mention API level instead (of SDK version) for clarity and perhaps link to that page in the comment above?

Reply:
My intention is for you to include the link itself into the script, but it's up to you, actually.

Reply:
For completeness sake, I've now included a link to the page as well.